src/main/java/omero/model/ElectricPotentialI.java

```diff
@@ -784,13 +784,17 @@ public ElectricPotentialI(ElectricPotential value, String target) throws BigResu
                         value.getValue(), value.getUnit(), target));
             }
             double orig = value.getValue();
-            BigDecimal big = conversion.convert(orig);
-            double converted = big.doubleValue();
-            if (Double.isInfinite(converted)) {
-                throw new BigResult(big,
-                        "Failed to convert " + source + ":" + target);
+            double converted = orig;
+            if (Double.isInfinite(orig)) {
+                // Do nothing. Use orig
+            } else {
+                BigDecimal big = conversion.convert(orig);
+                converted = big.doubleValue();
+                if (Double.isInfinite(converted)) {
```

Comment:
Is this still plausibly true?

Reply:
https://docs.oracle.com/javase/7/docs/api/java/math/BigInteger.html#doubleValue()

> if this BigInteger has too great a magnitude to represent as a double, it will be converted to Double.NEGATIVE_INFINITY or Double.POSITIVE_INFINITY as appropriate

Reply:
Aha, thank you, makes sense. :+1: